piwebapi_samples/Data Analysis/PIUploadUtility/PIUploadUtility/PIUploadUtility/PIUploadUtility.csproj

```diff
@@ -0,0 +1,58 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Did we consider making this a .NET Core app? Those are more common among our samples. I don't think it's necessary to switch at this stage though.

Reply:
I did not consider making this a .NET Core app. This was a last minute addition to the project.